docs/_docs/user-interface/formula.md

```diff
@@ -6,15 +6,11 @@ permalink: /docs/formula/
 ### Formula
 
 You can use this step to create a column as a formula based on other columns.
-A column can be referenced by its name without quotes or any other escaping
-character.
+A column can be referenced by its name without quotes unless they include
+whitespaces, in such a case you need to use brackets '[]' (e.g. \[myColumn]).
```

Comment:
```suggestion
white-spaces, in such a case you need to use brackets '[]' (e.g. `[myColumn]`).
```

Reply:
`white-spaces` is a CSS property ;) https://developer.mozilla.org/fr/docs/Web/CSS/white-space
`whitespace` is right if we choose to believe in Wikipedia: https://en.wikipedia.org/wiki/Whitespace_character